internal/controller/teleport.go

```diff
@@ -7,23 +7,31 @@ import (
 	"github.com/kintohub/kinto-cli/internal/utils"
 )
 
-func (c *Controller) Teleport() {
+//Teleports / Port-Forwards remote services to the local machine.
+// Takes a flag as arg.
+// if flag is true, teleport will show all the available services inside a user account.
+// if false, it will only show the services belonging to the user's current local git repo
+func (c *Controller) Teleport(teleportAllFlag bool) {
+
 	utils.CheckLogin()
 	utils.StartSpinner()
 
-	utils.GetGitDetails()
-	envs, err := c.api.GetClusterEnvironments()
+	var clusterId string
+	var envDetails []api.EnvDetails
+	teleportableServices := 0
+	serialNumber := 1
+
+	if !teleportAllFlag {
+		utils.GetGitDetails()
```

Comment:
Have a couple of related comments about func name 

I was so confused by this, coz `GetGitDetails` returns a bool and you are just calling it not in a condition or not using the result, after reading it I understood, whenever you have a func that checks for something + could kill ur app, it would be better to add `OrDie` at the end, so u would name it `GetGitDetailsOrDie()`

The 2nd thing, I saw that this func returns bool, and i think you are not really getting the details u are just checking if this is a git repo, does it make sense to rename it to `IsGitRepoOrDie()` ?

(tip: generally when something returns a bool, its nice to start the func with `Is` or `Has`, will make it easy to understand)

Reply:
If `IsGitRepoOrDie()` is not right because this func is doing something diff/extra feel free to find a name that make sense, its up to u

Reply:
My bad, GetGitDetails was supposed to be in an IF block since it's needed for checking flag for teleport. I somehow missed it during testing.

And GetGitDetails does two things depending on no. Of parameters, if you pass no parameters it will just check that if you are in a local git repo or not while if you pass A repo url (that you can get from build config) to it as a param, it will compare that url with the local git repo url and return a book accordingly.
This was done with the intention to have a single function handling the git related tasks. I agree GetGitDetails might not be the best name for it since the two functions it performs are quite different